maps/southern_cross/southern_cross-5.dmm

```diff
@@ -1200,19 +1213,22 @@
 "zz" = (/obj/effect/floor_decal/spline/plain{dir = 9},/obj/item/weapon/beach_ball,/turf/simulated/floor/tiled/freezer,/area/surface/outpost/civilian/pool)
 "zA" = (/obj/effect/floor_decal/borderfloor{dir = 8},/obj/effect/floor_decal/corner/orange/border{dir = 8},/obj/machinery/power/apc{dir = 8; name = "west bump"; pixel_x = -24},/obj/structure/cable/blue,/obj/machinery/light_switch{pixel_x = -22; pixel_y = -10},/turf/simulated/floor/tiled,/area/surface/outpost/civilian/fishing)
 "zC" = (/obj/effect/floor_decal/steeldecal/steel_decals_central1{dir = 1},/turf/simulated/floor/tiled/monotile,/area/surface/outpost/main/gym)
+"zD" = (/obj/effect/floor_decal/borderfloor{dir = 1},/obj/effect/floor_decal/corner/yellow/border{dir = 1},/obj/machinery/computer/general_air_control/supermatter_core{input_tag = "rust_cooling_in"; name = "Engine Cooling Control"; output_tag = "rust_cooling_out"; sensors = list("rustengine_sensor" = "Engine Tempeture")},/turf/simulated/floor/tiled,/area/surface/outpost/engineering/reactor_smes)

```

Comment:
```
"zD" = (/obj/effect/floor_decal/borderfloor{dir = 1},/obj/effect/floor_decal/corner/yellow/border{dir = 1},/obj/machinery/computer/general_air_control/supermatter_core{input_tag = "rust_cooling_in"; name = "Engine Cooling Control"; output_tag = "rust_cooling_out"; sensors = list("rustengine_sensor" = "Engine Tempeture")},/turf/simulated/floor/tiled,/area/surface/outpost/engineering/reactor_smes)

```

Smol quip about the spelling of the tag here, ahem.

Reply:
Dyslexia strikes again